### DISCUSSION TITLE
Add `musl` detection in `postinstall.js`

### DISCUSSION THREAD

**Discussion by matteosacchetto** [biomejs/biome#946] [Ideas]
I am very happy that biome now is build also for the `musl` target.

Though, one thing which is missing is the correct detection of that target in the [postinstall script](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/scripts/postinstall.js)

I think we could use this discussion to keep track of the best way to add `musl` detection

Here is a discussion which could be helpful as a starting point: https://github.com/nodejs/node/pull/17993

**Comment by ematipico** [comment]
In case someone is interested, here's how it can be done

https://github.com/microsoft/vscode-ripgrep/pull/25

**Comment by Conaclos** [comment]
Implemented in #1067
